cmds/modules/noded/main.go

```diff
@@ -191,6 +192,39 @@ func action(cli *cli.Context) error {
 		return version, nil
 	})
 
+	bus.WithHandler("zos.system.diagnostics", func(ctx context.Context, payload []byte) (interface{}, error) {
```

Comment:
i don't like that the implementation is like this in the main file. The WithHandler can still take a function that is at least defined separately as it's own function (not anonymous) and preferably in a separate file